doc/source/whatsnew/v0.21.0.txt

```diff
@@ -138,6 +138,16 @@ Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 
+.. _whatsnew_0210.api_breaking.Period:
+
+pd.Period objects are now immutable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:class:`Period` will now raise an ``AttributeError`` when a user tries to
+assign a new value to the `ordinal` or `freq` attributes (:issue:`17116`).
+=======
```

Comment:
extra line here. IIRC I told you to make a sub-section, but let's move this back to just a single line (still in API breaking change)